src/components/Modal/index.tsx

```diff
@@ -82,6 +82,7 @@ interface ModalProps {
   minHeight?: number | false
   maxHeight?: number
   initialFocusRef?: React.RefObject<any>
+  className?: string
```

Comment:
to extend uniswap `Modal` component via styled

Reply:
Shouldn't this be on a Mod file?

Reply:
i made a judgement call not to